app/views/shared/_contact_panels.html.erb

```diff
@@ -3,7 +3,7 @@
 <%= render 'shared/pas_chat_banner' if pensions_and_retirement_page? %>
 
 <div class="l-constrained">
-  <div class="l-contact-panels t-contact-panels <%= 'l-contact-panels--no-border-top' if contact_panels_border_top %> <%= 'l-contact-panels--homepage' if contact_panels_homepage %>">
+  <div class="l-contact-panels t-contact-panels <%= 'l-contact-panels--no-border-top' if contact_panels_border_top %> <%= 'l-contact-panels--homepage' if contact_panels_homepage %>" data-dough-contact-panels>
```

Comment:
This markup really needs to be refactored. The erb code can be extracted abstracted into the presenter, happy to help with this refactoring. I understand that this markup was written previously and strictly speaking not part of this pr; but your changes highlight its need for refactoring.

Reply:
I agree but I'm not sure if I should be the one to do it, I've added this markup to use it as a selector in javascript. These panels are also going to be redone in ticket 10511.

Reply:
@andreferreiradlw I'll have a look to see how easy it is (or not)

Reply:
@andreferreiradlw I think pr #2126 is a small improvement, wdyt?

Reply:
@margOnline it looks better then before in my opinion :)